### PR TITLE
Qa8 1292 anna

### PR DESCRIPTION
**Pull request by AnnaBondaruck**
Please review! I still need to work on TC-181, TC-182.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "qa8-mlh",
```
Comment (dalaran78) [inline package-lock.json:2]: No need to push package-lock.json file to main branch. It will affect to setup settings for all project.

**Review by dalaran78 — DISMISSED** [review]

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]
Please re-upload your file here without package-lock.json.

File: helpers/methods.js [hunk 2]
```diff
@@ -81,6 +81,26 @@ function uploadImage (image) {
...
+  $(sel.name).setValue(name);
+  $$(sel.radioButtons)[gender].click();
+  $(sel.age).setValue(age);
+  $(sel.storyType).click();
+  $$(sel.storyList)[story].click();
+  $(sel.submit).click();
+  $(sel.tryAgain).click();
+}
+
+function createDispAfteReset (name, gender, age, story) {
```
Comment (MariiaSosedova) [inline helpers/methods.js:94]: Do we really need two function with the same body?
Maybe use only one of then, just need to rename it (exp. input4FieldsSubTryagain or something like that?)

File: test/extended/reset.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import {inputValues4, isResetDisabled, createDispAfteReset} from '../../helpers/methods';
+import exp from '../../data/expected.json';
+
+describe('RESET TESTS SUITE', function () {
+
+    before('Open app', function () {
+        browser.url('');
+    });
+
+    describe('Reset button placeholder = "Try again!"', function () {
```
Comment (MariiaSosedova) [inline test/extended/reset.js:12]: please add the N of TC in accordance with the excel file

File: test/extended/reset.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            let submitBtn = $(sel.submit).isEnabled();
+            expect(submitBtn).toEqual(false);
+        });
+
+        it('TC-184 The "Create!" button is displayed after a user presses "Try again!"', function () {
+            createDispAfteReset(name.default, gender.it, age.default, story.comedy);
+            let submitBtn = $(sel.submit).isDisplayed();
+            expect(submitBtn).toEqual(true);
+        });
+
```
Comment (MariiaSosedova) [inline test/extended/reset.js:39]: please delete empty line

File: test/extended/reset.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    });
+    afterEach('Refresh page', () => {
```
Comment (MariiaSosedova) [inline test/extended/reset.js:41]: and here add empty line) - before line 41

**Review by MariiaSosedova — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (AnnaBondaruck) [reply package-lock.json:2]: Done

**Comment by MariiaSosedova** [comment]
this branch in not actual anymore. There is a new one with all changes

**Pull request closed without merging**